### DISCUSSION TITLE
Welcome to openTenancy! An introduction and questions thread

### DISCUSSION THREAD

**Discussion by amyconroy** [open-tenancy/opentenancy_law#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the openTenancy community. We hope that you:
  * Introduced yourself! We wanted openT to be a community 
  * Ask questions (we don't bite!)
  * Share ideas about the future of openT
  * Engage with other members
  * Chat about anything and everything!
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself!

**Comment by amyconroy** [comment]
# Hello everyone! I'll kick this off: 

## My name is Amy

I am one of openTenancy's co-founders, and a maintainer of this repository. 

## I have my LLB and MSc Computer Science Degrees

I have a nack for Legal Tech, and love everything about it. I work at Avvoka, a legal tech startup, and have also published a paper as part of the JURIX 2020 conference proceedings on my research on creating an automatic case summarisation system. 

## Feel free to ask any questions!

Happy to have a chat, and get this discussion going!

**Comment by a-shmyg** [comment]
## Hi everyone, just to continue the discussion board:
### I'm Ana!
The second openTenancy co-founder, and also a maintainer of the repo. I'm a Marine Biologist turned Computer Scientist, and having great fun with working using the Docassemble platform! If you're curious about the tech used to build openTenancy, give me a shout!

**Comment by Nkat99** [comment]
Hiya Amy and Ana. I'm Noni. Pleased to make your acquaintance. Obviously you are both awesome! I came across this project on Linkedin. 
I'm from Australia. I have LLB and B Comp Sci degrees. I'm a graduate lawyer. I work for a firm that specialises in real estate law (not property law or conveyancing, but many clients are in the real estate industry). So I feel like this project also ties in with some of the issues I have picked up on in my legal practice especially with the COVID measures brought in this year protecting tenants from eviction. And in this role I also worked on building simple bots to help people navigate these rules. 
Is it correct that most of the repair topics are done now? 
I would like to have a go at the topic of Rent Increases.  I feel that my initial hurdle will be researching the unfamiliar UK legal system .
Edited to add my name :)

**Reply by amyconroy** [reply]
So excited to have you on board Noni! 

Most of the repairs are there (I spent a few hours on Friday bashing out some of the flowcharts!). we are more than happy for you to take the lead for rent increases. We've left some links such as citizens advice bureau, tenants voice, as well as gov.uk - despite having studied in England I still need to refresh myself with these sites (and even learn things fresh!). Feel free to shout if you need a hand finding a reputable source/need some signposting. 

for now, I will move rent increases into ongoing with a note that you're giving those ago - obviously don't feel as though you need to complete it all so feel free to pass on the torch whenever you'd like by making a PR :)

**Comment by a-shmyg** [comment]
Hi Noni! 

Thank you so much for helping us out. Also another point to clarify - within the topic choices (ie for repairs, rent increases) it makes sense to keep things small and modular with sub-topics, so feel free to split it up however you like (and as Amy mentioned don't feel the need to complete it all, only as much as you have time for!). Good to have you on board!

**Comment by JohnG60** [comment]
Hi, I'm a lawyer in Tucson, AZ that heard Amy(?) on a podcast. She mentioned wanting to expand in the future. Would like to become relevant and am willing to assist. Not much coding knowledge.
